python/mlc_chat/compiler/model/model.py

```diff
@@ -122,6 +123,19 @@ class Model:
             "group-quant": gpt_bigcode_quantization.group_quant,
         },
     ),
+    "phi-msft": Model(
```

Comment:
i'm just curious why its name is `phi-msft` rather than `phi`?

Reply:
In hf repo, its `model_type` in `config.json` is `phi-msft`